src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -102,4 +108,68 @@ export default class DynamoDbParamBuilder {
             Item: DynamoDBConverter.marshall(newItem),
         };
     }
+
+    static buildPutCreateExportRequest(
+        jobId: string,
+        initiateExportRequest: InitiateExportRequest,
+        stepFunctionExecutionArn: string,
+    ) {
+        const item = {
+            jobId,
+            jobOwnerId: initiateExportRequest.requesterUserId,
+            exportType: initiateExportRequest.exportType,
+            groupId: initiateExportRequest.groupId ?? '',
+            outputFormat: initiateExportRequest.outputFormat ?? 'ndjson',
+            since: initiateExportRequest.since ?? '1800-01-01T00:00:00Z', // Default to a long time ago in the past
+            type: initiateExportRequest.type ?? '',
+            transactionTime: initiateExportRequest.transactionTime,
+            s3PresignedUrls: [],
+            stepFunctionExecutionArn,
+            jobStatus: 'in-progress',
+            jobFailedMessage: '',
+        };
+        return {
+            TableName: EXPORT_REQUEST_TABLE,
+            Item: DynamoDBConverter.marshall(item),
+        };
+    }
+
+    static buildQueryExportRequestJobStatus(jobStatus: ExportJobStatus) {

```

Comment:
Can further optimize by adding projections; in our case we only want requestorId and job status

https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/blob/mainline/src/dataServices/dynamoDbParamBuilder.ts#L67

Reply:
added